src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3010,6 +3010,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       _GenerateCompileInputs;
       BeforeCompile;
       _TimeStampBeforeCompile;
+      _GenerateCompilerDependencyCache;
```

Comment:
Why add this here instead of doing a `BeforeTargets="CoreCompile"`? Just for relative ordering?